src/main/tools/eclipse/freelibrary-cleanup.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
The Eclipse and Travis tools don't need to be checked into GitHub, just the PMD and Checkstyle ones so you can include the eclipse and travis paths in the .gitignore: e.g., https://github.com/UCLALibrary/fester/blob/main/.gitignore#L14

Reply:
@DRickard You marked this as resolved but I still see the `src/main/tools/travis` and `src/main/tools/eclipse` files in this PR. I saw they were added to the .gitignore but you'll need to remove them from the PR too.